### PR TITLE
add jeroenrinzema/psql-wire

### PR DESCRIPTION
**Pull request by jeroenrinzema**
> Please check if what you want to add to `awesome-go` list meets [quality standards](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#quality-standards) before sending pull request. Thanks!

**Please provide package links to:**

- repo link (github.com, gitlab.com, etc): https://github.com/jeroenrinzema/psql-wire
- pkg.go.dev: https://pkg.go.dev/github.com/jeroenrinzema/psql-wire
- goreportcard.com: https://goreportcard.com/report/github.com/jeroenrinzema/psql-wire
- coverage service link ([codecov](https://codecov.io/), [coveralls](https://coveralls.io/), [gocover](http://gocover.io/) etc.) n/a:

**Note**: _that new categories can be added only when there are 3 packages or more._

**Make sure that you've checked the boxes below that apply before you submit PR.**
_Not every repository (project) will require every option, but most projects should. Check the Contribution Guidelines for detials._

- [x] The package has been added to the list in alphabetical order.
- [x] The package has an appropriate description with correct grammar.
- [x] As far as I know, the package has not been listed here before.
- [x] The repo documentation has a pkg.go.dev link.
- [x] The repo documentation has a coverage service link.
- [x] The repo documentation has a goreportcard link.
- [x] The repo has a version-numbered release and a go.mod file.
- [x] I have read the [Contribution Guidelines](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#contribution-guidelines), [Maintainers Note](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#maintainers) and [Quality Standards](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#quality-standards).
- [x] The repo has a continuous integration process that automatically runs tests that must pass before new pull requests are merged.
- [x] The authors of the project do not commit directly to the repo, but rather use pull requests that run the continuous-integration process.

Thanks for your PR, you're awesome! :+1:

### CODE REVIEW

**Comment by avelino** [comment]
Thank you for contributing with [awesome-go](https://github.com/avelino/awesome-go), we will revise your contribution as soon as possible.

Automation body links content check:
- godoc.org or pkg.go.dev: True
- goreportcard.com: True
- coverage: True

> your project is about to be approved, it's under revision, _it may take a few days_

**Comment by jeroenrinzema** [comment]
Would someone be able to take a look at this PR?

File: README.md [hunk 1]
```diff
@@ -1855,6 +1855,7 @@ _Libraries for working with various layers of the network._
 - [panoptes-stream](https://github.com/yahoo/panoptes-stream) - A cloud native distributed streaming network telemetry (gNMI, Juniper JTI and Cisco MDT).
 - [peerdiscovery](https://github.com/schollz/peerdiscovery) - Pure Go library for cross-platform local peer discovery using UDP multicast.
 - [portproxy](https://github.com/aybabtme/portproxy) - Simple TCP proxy which adds CORS support to API's which don't support it.
+- [psql-wire](https://github.com/jeroenrinzema/psql-wire) - PostgreSQL server wire protocol.
```
Comment (yassinebenaid) [inline README.md:1858]: Can you rephrase the description. 
may be:
>PostgreSQL server wire protocol. Build your own server and start serving connections.

**Review by yassinebenaid — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Morton22) [reply README.md:1858]: Who the hell are you trying to steal my assets from me?????

Comment (Morton22) [reply README.md:1858]: > Can you rephrase the description. 
> 
> may be:
> 
> >PostgreSQL server wire protocol. Build your own server and

**Review by yassinebenaid — APPROVED** [review]
